cclib/bridge/cclib2horton.py

```diff
@@ -72,10 +72,8 @@ def makehorton(ccdat):
             attributes["orb_alpha"] = ccdat.mocoeffs[0]
             if len(ccdat.mocoeffs) == 2:
                 attributes["orb_beta"] = ccdat.mocoeffs[1]
-        if hasattr(ccdat, "coreelectrons") and isinstance(
-            ccdat.coreelectrons, numpy.ndarray
-        ):  # type-checked
-            attributes["pseudo_numbers"] = ccdat.atomnos - ccdat.coreelectrons
+        if hasattr(ccdat, "coreelectrons"):
```

Comment:
Could we rename `ccdat` to just `data`?

Reply:
Yes, I've just renamed it to `data`. Thanks!